setup.cfg

```
@@ -48,7 +48,7 @@ package_dir =
 # For more information, check out https://semver.org/.
 install_requires =
     aiohttp
-    aleph-sdk-python==0.7.0
+    aleph-sdk-python>=0.8.0
```

It should rather be
```suggestion
    aleph-sdk-python~=0.8.0
```
otherwise we get the same problem at some point like with `aleph-client` on v0.3.x and v0.4.x